themes/finna2/templates/ajax/feed-carousel-item.phtml

```diff
@@ -1,6 +1,6 @@
 <!-- START of: finna - ajax/feed-carousel-item.phtml -->
 <?php
-  $clickableElement = !$this->touchDevice && !isset($this->linkText);
+  $clickableElement = !$this->touchDevice && !$hasLinkText;
```

Comment:
Tämä pommittaa, jos on jossain kustomoitu feed-carousel.phtml, josta puuttuu hasLinkText-parametri. Laita `!$this->hasLinkText`.